### ISSUE TITLE
a love letter to patching

### ISSUE THREAD

**Issue by michaelpalumbo** [dispersionlab/gitshow#4]
*note* am reposting this message of mine from our private thread because I want it to be public: 

> I began working on an instrument on Saturday evening. In honesty it has been quite some time since I afforded myself an evening of patching for the sake of simply creating something new but without overthinking its outward usefulness. It was really lovely. So far the result is a rather bizarre step-sequencer that uses the new multichannel objects in Max 8. I also tried to make commits to the repo more often than usual, which had an effect similar to journaling and offered moments of self-reflection on the work.
> 
> Some encouragement at this outset: I believe that the incremental nature of this project's cycles mean that you and I have freedom to put as little or as much time and energy into each of the cycles as you are comfortable with (particularly in view of your other commitments). While the goal of this cycle is for each of us to contribute a working version of a new instrument/score/recording, I leave the degree of complexity up to each of you, such that even a slightly modified .maxhelp file (for example) may be sufficient towards this end.
> 
> **TL;DR** prioritize submitting a working version over something that takes more time/energy than you have at the moment. Trust that in the ensuing weeks your instruments will continue to develop in curious and unexpected ways. Thanks for your time, and happy creating!
